WookieeMatt/Essence20
# Initiative rolls trigger roll dialog; modifier input; migration

Closes https://github.com/WookieeMatt/Essence20/issues/252

In this PR:
- The roll dialog now appears when initiative is rolled
- Storing the initiative formula on each actor so that `_getInitiativeFormula()` can access it
- Adding an input for initiative static modifier
- Deleting `templates/actor/parts/actor-zord-skills.hbs` because it isn't being used anywhere

Testing:

- 🚨 PLEASE TEST ON AN E20 WORLD YOU DON'T CARE ABOUT BREAKING 🚨
- Before pulling down this branch, ensure you have PCs created in your world. Make at least one with a non-d20 initiative.
- Pull down this branch, `initiative-roll-dialog`
- Be a pal and open devtools by pressing F12 before launching the test world. If something goes wrong, the logs could be helpful.
- Launch the test world
- You should see messages about the migration starting and eventually completing
- Your PCs' initiatives should be the same as before
- Rolling initiative should prompt you with the dialog
- Dialog functionality works as expected, including auto-snag for d20
- Initiative modifier input works as expected
